websites/docs-smoke-test/src/code-examples/example.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
We should also include an example for `curl`

Reply:
I'm not very familiar with curl. is the file extension ".curl"?

Reply:
I guess... https://filext.com/file-extension/CURL

We will have examples in curl (especially for the API requests), so we need to support this as well.

Reply:
curl is not a common file extension (filext.com is an autogenerated SEO site to pull traffic) but for practical reasons we can agree to allow `*.curl` files, map it to the bash highlighter and call it "cURL" in the UI  (--> multiline PR is related).   That would allow us to selectively choose whether the user sees "cURL" instead of Terminal / Shell / Console which is what the code actually is.